packages/web/src/scss/components/Skeleton/README.md

```diff
@@ -0,0 +1,66 @@
+# Skeleton
+
```

Comment:
it would be nice to mention the correct aria-* attributes on the wrapper. What do you think?